charts/logging-operator/templates/userrole.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.rbac.enabled }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "logging-operator.fullname" . }}-user
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+rules:
+- apiGroups:
+  - logging.banzaicloud.io
+  resources:
+  - flows
+  - outputs
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - logging.banzaicloud.io
+  resources:
+  - flows/status
+  - outputs/status
+  verbs:
+  - get
+- apiGroups:
+  - logging.banzaicloud.io
+  resources:
+  - syslogngflows
+  - syslogngoutputs
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - logging.banzaicloud.io
+  resources:
+  - syslogngflows/status
+  - syslogngoutputs/status
+  verbs:
+  - get
```

Comment:
Is `watch` a verb that applies to status subresources?

Reply:
I don't think so. Also I'm not entirely sure, why we need get on status. Do we need that at all?

Reply:
You mean the syslogng resource or all resources in general? Otherweise the user can't get access to the status.

Reply:
I mean if you have get access on a resource, you should have get access on the subresource as well.

Reply:
Yes, make sense. Tested and the status is editable in any way, but the Operator sets back.